optaplanner-quarkus-bom/pom.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.optaplanner</groupId>
+    <artifactId>optaplanner-parent</artifactId>
+    <version>8.10.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>optaplanner-quarkus-bom</artifactId>
```

Comment:
Naming is not perfect; it's definitely not just OptaPlanner and Quarkus dependencies. However, it's what the Quarkus platform member BOM should be.

Reply:
IMO, the naming is fine. In addition, you are importing the `quarkus-bom` which includes everything necessary for building and testing applications. We could say, it's in the line with how it's done in Quarkus itself.